metadata-ingestion/src/datahub/ingestion/source/tableau/tableau.py

```diff
@@ -2796,6 +2864,18 @@ def emit_workbook_as_container(self, workbook: Dict) -> Iterable[MetadataWorkUni
             tags=tags,
         )
 
+        if (
+            self.config.access_role_ingestion
+            and self.config.access_role_ingestion.enable_workbooks
+        ):
+            logger.info(f"Ingest access roles of '{workbook.get(c.LUID)}'")
```

Comment:
make it debug and log it as "workbook-id={....}"

Reply:
Changed it. Do you want any other log messages to be debug?